### DISCUSSION TITLE
infrastructure/synology/disable-dms-listening-on-80-443-ports/

### DISCUSSION THREAD

**Discussion by giscus** [fire1ce/3os.org#89] [General]
# infrastructure/synology/disable-dms-listening-on-80-443-ports/

Free 80,443 ports on Synology NAS for other applications.

https://3os.org/infrastructure/synology/disable-dms-listening-on-80-443-ports/

**Comment by julichan** [comment]
If you have a domain, you can reverse proxy with synology dsm without any scripts....
just set Config Panel > External Access > Advanced to the following:
 - Hostname: intranet.mydomain,com (any subdomain will do actually)
 - DSM Http: something else than 80 (5080 for exemple)
 - DSM Https: something else than 443 (5043 for exemple)
 
Then the reverse proxy will allow you to reverse proxy on port 80/443 as long as you specify a domain other than the one specified above, including the root domain,
